napari/_tests/test_examples.py

```diff
@@ -1,12 +1,15 @@
 import os
 import runpy
 from pathlib import Path
-from qtpy import API_NAME
+
 import pytest
+from qtpy import API_NAME
 
 import napari
 from napari.utils.notifications import notification_manager
 
+from .utils import slow
```

Comment:
```suggestion
from napari._tests.utils import slow
```

Just flagging this because it's the only place you used a relative import.

Reply:
oops, thanks.